Cargo.toml

```diff
@@ -12,12 +12,11 @@ description = "The rust language implementation of Raft algorithm."
 categories = ["algorithms", "database-implementations"]
 
 [dependencies]
-flat_map = "0.0.4"
-fnv = "1.0"
 log = "0.3"
 protobuf = "1.2"
 quick-error = "0.2"
 rand = "0.4"
+fxhash = "0.2.1"
```

Comment:
Hm, this library has has an open PR since December that hasn't been addressed: https://github.com/cbreeden/fxhash/pull/3. It does appear to be used in Firefox though. The code seems good overall.